src/content/blog/whats-new-october.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: "What's new in Astro - October"
+description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
+publishDate: "October 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-october/head.webp"
+socialImage: "/src/content/blog/_images/whats-new-october/og.webp"
+lang: "en"
+---
+
+## Astro releases
+
+Astro's development keeps marching forward, with several notable releases in October:
+
+- Astro 3.2: This update gives you more control over the browser history stack, enables page navigation with View Transitions, includes a route announcer for screen readers, and integrations can now dynamically add other integrations. [Learn more](/blog/astro-320/).
+- Astro 3.3: This release introduces an experimental `<Picture />` component, enhances syntax highlighting, brings package provenance, and adds various quality-of-life improvements. [Read more](/blog/astro-330/).
+- Astro 3.4: Introduction Page Partials, along with improvements in image optimization and a first preview of an experimental dev overlay. [Discover more](/blog/astro-340/).
+
+See the `#announcements` channel in Discord for all releases across the Astro repos, including Starlight, the Astro VS Code extension, compiler, packages and integrations! Follow [Astro on GitHub](https://github.com/withastro/) for all updates.
+
+## Hacktoberfest
+
+[Hacktober](https://hacktoberfest.com/) is officially over now! Thank you all for your amazing contributions and PRs! Here are some of our statistics:
+
+- 2 Astro core PRs
+- 49 Starlight PRs
+- 62 Docs PRs
+
+## Docs
+
+We extended the tutorial with some requested features!
+
+- [Content Collections](https://docs.astro.build/en/tutorials/add-content-collections/)
+- [View Transitions](https://docs.astro.build/en/tutorials/add-view-transitions/)
+
+"Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump in a shared CodeSpace or HackMD document and everyone can participate. This months's activities included:
+
+- Live reviewing and testing the two new mini-tutorials
+- Improved our docs PR template and created a new GitHub action to welcome first-time contributors
+- All hands on deck reviewing translation submissions using the Astro Docs checklist to get everyone's `hacktoberfest-accepted` PRs merged
+
+## Community
+
+### Community Awards September
+
+Each month, Astro recognizes contributors from around the community, supported by [Astro's Open Collective](https://opencollective.com/astrodotbuild) funding. Join us in celebrating the following people that made Astro great this month!
+
+- 🍾 [@itsmatteomanf](https://github.com/itsmatteomanf) - $250 - For initiative taken translating our docs into Italian.
+- 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in development on the Cloudflare adapter.
```

Comment:
This sentence sounds a bit hard to read and is also missing the "the" between "in" and "development". I've made a couple options you can choose from, any works for me, or you can also go with something different, the blog post is yours!
```suggestion
- 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leading the Cloudflare adapter development.
```
```suggestion
- 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in the development of the Cloudflare adapter.
```